### PR TITLE
Method Syntax Example

### LINKED ISSUE #6204: Method Syntax For Linq Examples
For queries like this group by example it would be handy to have the linq method syntax version alongside the query syntax version.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af60d154-0468-c450-ad3b-4f144ca6e693
* Version Independent ID: 646c3fab-389f-3a5e-4abe-86943fb85adc
* Content: [Perform a subquery on a grouping operation](https://docs.microsoft.com/en-us/dotnet/csharp/linq/perform-a-subquery-on-a-grouping-operation)
* Content Source: [docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md](https://github.com/dotnet/docs/blob/master/docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by matt-p-hansen**
Fixes #6204

### CODE REVIEW

File: docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md [hunk 1]
```diff
@@ -17,7 +17,11 @@ For more information about continuations, see [into](../language-reference/keywo
 > [!NOTE]
 > This example contains references to objects that are defined in the sample code in [Query a collection of objects](query-a-collection-of-objects.md).
 
-[!code-csharp[csProgGuideLINQ#23](~/samples/snippets/csharp/concepts/linq/how-to-perform-a-subquery-on-a-grouping-operation_1.cs)]  
+[!code-csharp[csProgGuideLINQ#23](~/samples/snippets/csharp/concepts/linq/how-to-perform-a-subquery-on-a-grouping-operation_1.cs)] 
+
+The query in the snippet above can also be written using method syntax. The code snippet below has a semantically equivalent query written using method syntax.
```
Comment (rpetrusha) [inline docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md]: nit: The code snippet below --> The following code snippet

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @matt-p-hansen, for adding an example and modifying the text of this article. I'll merge your example in the dotnet/samples repo momentarily. I also have one minor suggestion for a change in this article.

**Comment by rpetrusha** [comment]
Closing and reopening to begin a new internal build.

File: docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md (hunk 1 above)
Comment (matt-p-hansen) [reply docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md]: Thanks for the suggestion! I like that phrasing better.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @matt-p-hansen, for making the additional change, as well as for contributing to the dotnet/docs repo. We'll merge your PR now. Your example should be live on docs.microsoft.com in the next day or two.

**Pull request merged**